src/typings/electron-ipc.d.ts

```diff
@@ -0,0 +1,186 @@
+/**
+ * Custom declarations for Electron IPC topics.
+ */
+
+import Electron from 'electron';
+
+type RecursivePartial<T> = {
```

Comment:
Just repeating from an earlier comment:

Could you add a comment to the definition of RecursivePartial pointing to something like https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-8.html (or something less version-specific) and one line explaining what this does?